www/content/attributes/hx-trigger.md

```diff
@@ -155,3 +157,4 @@ The AJAX request can be triggered via JavaScript [`htmx.trigger()`](@/api.md#tri
 * `hx-trigger` is not inherited
 * `hx-trigger` can be used without an AJAX request, in which case it will only fire the `htmx:trigger` event
 * In order to pass a CSS selector that contains whitespace (e.g. `form input`) to the `from`- or `target`-modifier, surround the selector in parentheses or curly brackets (e.g. `from:(form input)` or `from:closest (form input)`)
+* Reset triggers only on the second successive event (e.g. 2 clicks on reset button) when hx-trigger contains other triggers (e.g. hx-trigger="change, reset"). Add any delay to reset as a workaround (e.g. hx-trigger="change, reset delay:0.01s").  
```

Comment:
Unless I misunderstood, I don't think it is what is happening (or maybe is it simply a wording issue).
Take a look at [this JSFiddle](https://jsfiddle.net/pz8w31mL/):
- `reset` does trigger on the first click, you don't have to click 2 times to trigger it. The issue is that htmx builds its values set & fires its request _before_ the browser actually resets the form. So 1 click on the reset button already triggers htmx, and clicking twice would trigger 2 requests (one with the form not reset, then the form resets, then the second click sends a request with the form's reset values)
- This is due to the event listener's logic being called _before_ the browser's logic, as you may want to call `event.preventDefault` here to prevent the reset from happening at all
- Adding any delay as a workaround works, because then you let htmx fire its request & build its values set _after_ the browser has reset the form

Reply:
My bad, you're absolutely right. I've rewritten the proposed changes with your explanation in mind.